security/certificate_types_descriptions/machine-config-operator-certificates.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
+[id="cert-types-machine-config-operator-certificates"]
+= Machine Config Operator certificates
+include::_attributes/common-attributes.adoc[]
+:context: cert-types-machine-config-operator-certificates
+
+toc::[]
+
+== Purpose
+
+Machine Config Operator certificates are used to secure connections between the RHCOS nodes and the Machine Config Server.
```

Comment:
Since RHOS is the first usage , I would suggest expansion to Red Hat Enterprise Linux CoreOS (RHCOS). ref https://www.ibm.com/docs/en/ibm-style?topic=grammar-abbreviations